### ISSUE TITLE
libgcc_s_dw2-1.dll was not found

### ISSUE THREAD

**Issue by larrysharette** [fabiangreffrath/crispy-doom#570]
### Background

Version of Crispy Doom: 5.7.2

Operating System and version: Windows 10 1909

Game: Doom

Any loaded WADs and mods (please include full command line): None

### Bug description

Observed behavior: Clicking on crispy-doom.exe produces the following error.
```
The code execution cannot proceed because libgcc_s_dw2-1.dll was not found. Reinstalling the program may fix this problem.
```

Expected behavior: Doom starts up.

I downloaded version 5.7 and it is running just fine. I will stay with this version for now.

**Comment by fabiangreffrath** [comment]
@Zodomaniac what's going on here?

**Comment by Zodomaniac** [comment]
@fabiangreffrath , that's what happens if I combine your autobuilder's DLLs with my homebrew binaries. Should I re-upload the version I built, with 29 DLLs?

**Comment by fabiangreffrath** [comment]
If that's necessary to get the release in a usable state, yes.

**Comment by Zodomaniac** [comment]
I did that.

**Comment by fabiangreffrath** [comment]
Thanks!

**Comment by Zodomaniac** [comment]
@larrysharette if you want to try 5.7.2, please re-download it :)

**Comment by larrysharette** [comment]
@Zodomaniac I can confirm that its running now. Although its triggering windows defender and its quarantining the crispy-doom.exe. It isn't a huge issue because I can pull it from the ZIP folder but I thought I would let you guys know.

**Comment by fabiangreffrath** [comment]
Should be fixed with the 5.8.0 release.
